### DISCUSSION TITLE
What explains the difference between *colour* and *Wikipedia* CIELUV chromaticity diagram?

### DISCUSSION THREAD

**Discussion by danwahl** [colour-science/colour#1284] [Q&A]
### Question

Probably a noob question, but I'm curious why the CIELUV diagram generated by colour (`plot_chromaticity_diagram_CIE1976UCS()`) seems subtly different from the one on [Wikipedia](https://en.wikipedia.org/wiki/CIELUV) (note the u' coordinate of the reference white point):

![cieluv-colours](https://github.com/user-attachments/assets/ecc30bd1-5417-4c0b-8f9b-161eed3be1de)

![cieluv-wiki](https://github.com/user-attachments/assets/f71186ed-7781-49e6-9d13-0c3e912d9e46)

**Comment by thomasmansencal** [comment]
Hi @danwahl,

This is a great question to which I have no immediate answer! The whitepoint we use is that of our plotting routines working space: https://github.com/colour-science/colour/blob/e21773141a9726dc5c68777a4d38bdc985382132/colour/plotting/common.py#L170

It defaults to sRGB thus the whitepoint is D65.

I overlaid the Wikipedia on top of ours and I suspect that there is a rendering issue:

![image](https://github.com/user-attachments/assets/93a6439e-4d32-4fd1-82ef-b697e1477498)


Cheers,

Thomas
